src/frequenz/channels/experimental/_pipe.py

```diff
@@ -25,17 +25,27 @@ class Pipe(typing.Generic[ChannelMessageT]):
 
     Example:
         ```python
-        from frequenz.channels import Broadcast, Pipe
+        from contextlib import closing, aclosing, AsyncExitStack
+        from frequenz.channels import Broadcast, Pipe, Receiver
 
-        channel1: Broadcast[int] = Broadcast(name="channel1")
-        channel2: Broadcast[int] = Broadcast(name="channel2")
+        # Channels, receivers and Pipe are in AsyncExitStack
+        # to close and stop them at the end.
 
-        receiver_chan1 = channel1.new_receiver()
-        sender_chan2 = channel2.new_sender()
+        async with AsyncExitStack() as stack:
+            source_channel = await stack.enter_async_context(
+                aclosing(Broadcast(name="source channel")))
+            source_receiver = await stack.enter_async_context(
+                closing(source_channel.new_receiver()))
```

Comment:
Does this really works? `AsycnExitStack()` also provides the methods provided by `ExitStack`, so you should probably use `enter_context()` here that the context manager is *sync*.
```suggestion
            source_receiver = await stack.enter_context(
                closing(source_channel.new_receiver()))
```

Reply:
Yea sorry I assumed the code in `Example` is executed. (because they are showed in the pytest summary as tests.
I just found out only imports are checked.

Reply:
`pylint` is also ran, but the code itself isn't, nor `mypy` or other *lint* tools.